content/en/docs/pipelines/installation/overview.md

```diff
@@ -169,6 +180,13 @@ Upgrade Support (**Alpha**)
 
 For upgrade via reinstallation (with existing data) instructions, refer to [Upgrading AI Platform Pipelines](/docs/gke/pipelines/upgrade/#ai-platform-pipelines).
 
+Google Cloud Integrations
+:
+  * You can deploy AI Platform Pipelines on [Google Cloud Console UI](https://console.cloud.google.com/marketplace/details/google-cloud-ai-platform/kubeflow-pipelines).
+  * A Kubeflow Pipelines public endpoint with auth support is **auto-configured** for you.
+  * (Optional) You can choose to persist your data in Google Cloud managed storages (Cloud SQL and Cloud Storage).
+  * You can [authenticate to Google Cloud with Compute Engine default service account](/docs/gke/pipelines/authentication-pipelines/#compute-engine-default-service-account). It is easy to set up, but not suitable if you need workload permission separation.
```

Comment:
Suggestion (as before): 
- Rephrase for a11y (~~easy~~ if possible)

```diff
- It is easy to set up, but not suitable if you need workload permission separation.
+ However, this method may not be suitable if you need workload permission separation.
```

(or something among those lines)